ui/cypress/integration/volume/createvolume.spec.js

```diff
@@ -0,0 +1,70 @@
+beforeEach(() => {
+  cy.setupMocks();
+  cy.login();
+  cy.visit('/volumes/createVolume');
+});
+
+const VOLUME_NAME = 'test-volume-sparse';
+const STORAGECLASS = 'metalk8s';
+const VOLUME_LABEL_NAME = 'kubernetes.io/name';
+const VOLUME_LABEL_INVALID = 'kubernetes.io&/name';
+const VOLUME_LABEL_VALUE = 'test';
+const VOLUME_TYPE = 'sparseLoopDevice';
+const VOLUME_SIZE = '1 GiB';
+
+// interaction test for creating volume
+describe('the create button in create volume page', () => {
+  it('should be enabled and the Add button should be disabled when no labels is specified', () => {
+    cy.get('input[name=name]').type(VOLUME_NAME);
+
+    cy.get('.sc-select').eq(0).click();
+    cy.get('.sc-select__menu').eq(0).click();
+    cy.get('.sc-select').eq(1).click();
+    cy.get('.sc-select__menu')
+      .find(`[data-cy=storageClass-${STORAGECLASS}]`)
+      .click();
+    cy.get('.sc-select').eq(2).click();
+    cy.get('.sc-select__menu').find(`[data-cy="type-${VOLUME_TYPE}"]`).click();
+    cy.get('input[name=sizeInput]').type(VOLUME_SIZE);
+
+    cy.get('[data-cy="submit-create-volume"]').should('be.enabled');
+    cy.get('[data-cy="add-volume-labels-button"]').should('be.disabled');
```

Comment:
use cy.queryByRole('button', {name: /add/i})

Reply:
Just FYI in case you miss it:
https://github.com/testing-library/cypress-testing-library/releases/tag/v6.0.0
so should be: `  cy.findByRole('button', { name: /create/i })`